app/models/episode.rb

```diff
@@ -210,4 +210,14 @@ def media_files
   def audio_files
     media_files
   end
+
+  def set_adzerk_keyword
+    return unless published? && adzerk_keyword.nil?
+    identifiers = []
+    [:title, :published_at, :guid].each do |attr|
+      # Adzerk does not allow commas or colons in keywords
+      identifiers << self.send(attr).to_s.slice(0, 10).gsub(/[:,]/,'')
+    end
+    self.adzerk_keyword = identifiers.join('_')
```

Comment:
I think I would be tempted to put the date first, and I wonder if we need the dashes in it? (save 2 chars!)

Likewise with the guid: "6d6f619d-5052-4330-80b8-34f414d076f9" becomes "6d6f619d-5", could become "6d6f619d50" without the dash.

And maybe put those additional chars towards the title which is pretty short at just 10 chars, especially if there is an episode # up front?

`20170124_100 - This is a_6d6f619d50`

is that better to anyone else? Maybe we should check with Kali?